ignite/distributed/comp_models/base.py

```diff
@@ -132,9 +134,20 @@ def _apply_op(self, tensor: torch.Tensor, device: torch.device, fn: Callable, *a
             return tensor.to(device=tensor_device)
         return tensor
 
+    @overload
```

Comment:
Maybe, we can simply ignore type check on `_collective_op` and `broadcast` now and come back later ?

Reply:
Sure, I will try it out tomorrow.